npm/webpack-dev-server/src/helpers/createReactAppHandler.ts

```diff
@@ -77,6 +77,8 @@ function loadWebpackConfig (devServerConfig: WebpackDevServerConfig): Configurat
       webpackConfig = webpackConfig('development')
     }
 
+    delete webpackConfig.entry
```

Comment:
How come we need this? Seems unrelated to Angular. Drive by bug fix?

Reply:
Right: https://github.com/cypress-io/cypress/pull/22314/files#r927118026

This must have been a bit painful to figure out.

Reply:
Yes I had to refactor the entrypoints of other handlers to get this to work. I could have had an Angular specific hack here, but I wanted to minimize framework specific logic here as this is common to all.